src/App.js

```diff
@@ -7,22 +7,39 @@ class App extends React.Component {
   constructor() {
     super()
     this.state = {
-      movies: movieData.movies
+      movies: [],
+      error: ''
     }
     this.showMovieDetails = this.showMovieDetails.bind(this)
     this.goBack = this.goBack.bind(this)
   }
+  fetchAllMovies() {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => {
+      if(!response.ok) {
+        throw new Error('sorry try again')
+      } else {
+        return response.json()
+      }
+    })
```

Comment:
Looking good here! Typically, In React we move our fetch calls into another file, import them into the component that will use them, and fire them there. Consider moving this to a networkRequests.js file, stored in `src` :)